frontend/__tests__/src/components/metrics/ConfigStep/ConfigStep.test.tsx

```diff
@@ -125,12 +128,12 @@ describe('ConfigStep', () => {
     const requireDateSelection = within(getByRole('listbox'))
     fireEvent.click(requireDateSelection.getByRole('option', { name: VELOCITY }))
     fillBoardFieldsInformation()
-    fireEvent.click(getByText('Verify'))
+    fireEvent.click(getByText(VERIFY))
     fireEvent.click(getByText(CHINA_CALENDAR))
 
-    expect(queryByText('Verify')).toBeVisible()
+    expect(queryByText(VERIFY)).toBeVisible()
     expect(queryByText('Verified')).toBeNull()
-    expect(queryByText('Reset')).toBeNull()
+    expect(queryByText(REST)).toBeNull()
```

Comment:
`REST`? or `RESET`?
And have u confirmed the error message which is described in description(`it.....`)